### PR TITLE
Add lockfile schema

### PR DESCRIPTION
**Pull request by LeoDog896**
This adds support for `deno.lock` with autocomplete and JSON recognition

This also solves #749

### CODE REVIEW

File: schemas/lockfile.schema.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/denoland/vscode_deno/blob/main/schemas/lockfile.schema.json",
+  "title": "Deno Lock file",
+  "description": "An import map which is used to remap imports when modules are loaded.\n\nSee https://deno.land/manual@v1.6.0/linking_to_external_code/import_maps in the Deno documentation for more information.",
+  "type": "object",
+  "required": [
+    "version",
+    "remote"
+  ],
```
Comment (bartlomieju) [inline schemas/lockfile.schema.json:10]: This file needs to be updated

Comment (LeoDog896) [reply schemas/lockfile.schema.json]: Fixed description of both the root and the sub-property, version

File: schemas/lockfile.schema.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      ]
+    },
+    "remote": {
+      "description": "A map of remote URLs to their respective hashes.",
+      "type": "object",
+      "additionalProperties": {
+        "type": "string",
+        "description": "The hash as the value for the remote URL as the key."
+      }
+    }
```
Comment (bartlomieju) [inline schemas/lockfile.schema.json:30]: It's missing `npm` property that is used to store integrities of `npm` packages. You can see example lock file here: https://github.com/denoland/deno/blob/7b212bc5741ea0cfb6bd87c280be0c864cb32813/cli/tests/testdata/npm/lock_file/lock.json

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks for looking into this @LeoDog896!

**Comment by LeoDog896** [comment]
By the way, this does close #749

**Pull request merged**